cfgmassif.go

```diff
@@ -24,19 +26,57 @@ func cfgMassifReader(cmd *CmdCtx, cCtx *cli.Context) error {
 		}
 	}
 
-	if cmd.reader == nil {
-		if err = cfgReader(cmd, cCtx); err != nil {
-			return err
-		}
-	}
-
-	massifReader := massifs.NewMassifReader(logger.Sugar, cmd.reader, massifs.WithoutGetRootSupport())
-	cmd.massifReader = massifReader
 	cmd.massifHeight = uint8(cCtx.Uint("height"))
 	if cmd.massifHeight == 0 {
 		cmd.massifHeight = defaultMassifHeight
 	}
 
+	localLog := cCtx.String("data-local")
+	remoteLog := cCtx.String("data-url")
+
+	if localLog != "" && remoteLog != "" {
+		return fmt.Errorf("can't use data-local and data-url at the same time")
+	}
+
+	if localLog == "" && remoteLog == "" {
+		// if we had no url and no local data supplied we try STDIN
```

Comment:
Might become evident later in review, but how does this interact with commands that take events on stdin?